public/index.html

```diff
@@ -36,6 +38,8 @@
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.6/js/materialize.min.js"></script>
     <!-- APP -->
     <script type="text/javascript" src="app/app.js"></script>
+    <!-- MouseWheel-->
+    <script type="text/javascript" src="lib/node_modules/angular-mousewheel/node_modules/hamsterjs/hamster.js"></script>
```

Comment:
Based on the HamsterJS documentation (and where HamsterJS ended up when I installed it locally), this script tag is incorrect.  Locally I had to change this to:  `<script type="text/javascript" src="lib/bower_components/hamsterjs/hamster.js"></script>` in order to make this branch work.

Anyone else seeing this as an issue?